src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -84,31 +117,51 @@ export class GitHubApps {
 
 export function loadGitHubAppsFromEnvironment(env) {
   const configs = new GitHubAppConfigs();
-  let config;
-
-  if (env.GH_APP_IDENTIFIER && env.GH_APP_SECRET_KEY) {
-    // Collect config by (proleptic) envvar number. Once we have GH_APP_1_FOO
-    // this will make more sense. We'll collect stuff in config and then
-    // instantiate a GitHubApp once all config has been collected for each
-    // (once we've made a full pass through process.env).
-
-    config = configs.getOrCreate(1);
-    config.org = '__tmp_org_placeholder__';
-    config.auth = {
-      appId: Number(env.GH_APP_IDENTIFIER),
-      privateKey: env.GH_APP_SECRET_KEY.replace(/\\n/g, '\n'),
-    };
-    config.project = {
-      node_id: env.ISSUES_PROJECT_NODE_ID || 'PVT_kwDOABVQ184AOGW8',
-      product_area_field_id:
-        env.PRODUCT_AREA_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgJEBno',
-      status_field_id: env.STATUS_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgI_7g0',
-      response_due_date_field_id:
-        env.RESPONSE_DUE_DATE_FIELD_ID || 'PVTF_lADOABVQ184AOGW8zgLLxGg',
-    };
+
+  for (const [envvar, value] of Object.entries(env)) {
+    // Find app configuration in env, grouping by number (GH_APP_1_FOO).
+
+    const m = envvar.match(/^GH_APP_(\d+)_([A-Z_]+)$/);
+    if (m === null) {
+      continue;
+    }
+
+    const n = parseInt(m[1], 10);
+    const noop = (x) => x;
```

Comment:
nit: might be more accurate to name this as `identity_fn`?

Reply:
Eh, maybe. Since the function is called `mod` below as in `modify`, `noop` seems fine since it is in fact performing no modification operation. :)